FileCloner/Models/Constants.cs

```diff
@@ -27,8 +27,8 @@ public class Constants
     );
 
     // Icon Paths
-    public static readonly string LoadingIconPath = Path.GetFullPath(Path.Combine("..", "..", "..","..","FileCloner","Assets", "Images", "loading.png"));
-    public static readonly string FileIconPath = Path.GetFullPath(Path.Combine("..", "..", "..","..","FileCloner", "Assets", "Images", "file.png"));
+    public static readonly string LoadingIconPath = Path.GetFullPath(Path.Combine("..", "..", "..", "..", "FileCloner", "Assets", "Images", "loading.png"));

```

Comment:
I do not see any changes in the removed lines & added lines, is it change in formatting ?

Reply:
Yes, editor config warned that the spaces between consecutive ".." were not separated by a space.